server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/pageobjects/RuleDetails.java

```diff
@@ -19,14 +19,35 @@
  */
 package org.sonarqube.qa.util.pageobjects;
 
-import com.codeborne.selenide.Selenide;
-
 import static com.codeborne.selenide.Condition.text;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$$;
 
 public class RuleDetails {
 
+  public RuleDetails shouldHaveType(String type) {
+    $$(".coding-rules-detail-property").findBy(text(type)).shouldBe(visible);
+    return this;
+  }
+
+  public RuleDetails shouldHaveSeverity(String severity) {
+    $(".coding-rules-detail-property .icon-severity-" + severity.toLowerCase()).shouldBe(visible);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define the locale to be used in this String operation. ([squid:S1449](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1449))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fstas%2Fit-rules&issues=AWEuHWgSr8h_fVHZ4iWu&open=AWEuHWgSr8h_fVHZ4iWu)

[ISSUE_KEY]: AWEuHWgSr8h_fVHZ4iWu